astro.config.mjs

```diff
@@ -10,9 +10,9 @@ export default defineConfig({
     }),
     tailwind(),
     compress({
-      path: ['./build', './dist'],
-      logger: 0,
-      html: {
+      Path: ['./build', './dist'],
+      Logger: 0,
+      HTML: {
```

Comment:
```suggestion
      path: ['./build', './dist'],
      logger: 0,
      html: {
```

Reply:
In newer versions you need the options pascal cased, that's why they're `Path`, `Logger` and `HTML`.